app/services/katello/pulp3/smart_proxy_repository.rb

```diff
@@ -82,6 +82,21 @@ def orphan_repositories
         end
         repo_map
       end
+
+      def current_repositories_data(environment = nil, content_view = nil)
+        @pulp_repositories ||= smart_proxy.smart_proxy_sync_histories.map { |sync_history| sync_history.repository }
+
+        repos = Katello::Repository
+        repos = repos.in_environment(environment) if environment
+        repos = repos.in_content_views([content_view]) if content_view
+        puppet_envs = Katello::ContentViewPuppetEnvironment
+        puppet_envs = puppet_envs.in_environment(environment) if environment
+        puppet_envs = puppet_envs.in_content_view(content_view) if content_view
```

Comment:
we can just delete this

Reply:
the content view puppet environment code

Reply:
Deleted this for pulp3 service, still present for pulp service class..Not removing it there..